### DISCUSSION TITLE
how to hardcode the meterID?

### DISCUSSION THREAD

**Discussion by uriva9w** [bemasher/rtlamr#241] [Q&A]
Hi
I wand to hardcode the  MeterID, lest's say MeterID=154689790
I think this can be done in Flags.go...meterID = MeterIDFilter{make(UintMap)}?
Could you point me to the right place?
Thx

**Comment by bemasher** [comment]
You should use the filtered flag, or the RTLAMR_FILTERID environment variable to filter by meter id.
